src/HyperscriptLiteral.jl

```diff
@@ -1,13 +1,38 @@
 module HyperscriptLiteral
 
-using Hyperscript: Node, DEFAULT_HTMLSVG_CONTEXT
+# We call `tag` what Hyperscript.jl calls `Node`.
+# We call `type` or `tagtype` what Hyperscript.jl calls `tag`.
+# We call `props` what Hyperscript.jl calls `attrs`.
+using Hyperscript: AbstractNode, Node, DEFAULT_HTMLSVG_CONTEXT, context, tag, children, attrs
 
 export @htm_str
 
 macro htm_str(html)
-    esc(parse(html; interp=true))
+    htm = parse(html; interp=true)
+    esc(toexpr(htm))
```

Comment:
The returned expression should call `create_element`. This way, we may support other backends in the future.